src/types.rs

```diff
@@ -189,8 +200,8 @@ pub struct GetMerkleRes {
 pub struct HeaderNotification {
     /// New block height.
     pub height: usize,
-    #[serde(rename(serialize = "hex"))]
     /// Newly added header.
+    #[serde(rename(serialize = "hex", deserialize = "hex"), deserialize_with = "from_hex_header")]
```

Comment:
if both `serialize` and `deserialize` are the same I guess you could use [`rename = "hex"`](https://serde.rs/field-attrs.html#rename)

Reply:
fixed